spec/features/organizations_spec.rb

```diff
@@ -123,5 +122,24 @@
       expect(DeployService).not_to receive(:start)
       ComparisonService.new(project).refresh_comparisons
     end
+
+    it 'respects deploy blocks' do
```

Comment:
thoughts on a spec description that better deblurs what "respects" means? How about "does not kick off a deploy if there are any unresolved deploy blocks"?